packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -211,6 +225,16 @@ export class InputTimePicker
    */
   @Prop({ reflect: true }) form: string;
 
+  /**
+   * Formats the displayed time value in either 12 or 24 hour format.  Defaults to the `lang`'s preferred setting.
+   */
```

Comment:
WDYT about streamlining the use of "formats" in the doc, something like:

```suggestion
  /**
   * Displays the component's `value` in either 12 or 24 hour format.  Defaults to the `lang`'s preferred setting.
   */
```

Reply:
"Displays" is a much better word, great suggestion!